core/src/test/resources/application.properties

```diff
@@ -1,50 +1,44 @@
 # Datasource
-spring.datasource.url=jdbc:postgresql://localhost:5432/test
-spring.datasource.username=${DATASOURCE_USER}
-spring.datasource.password=${DATASOURCE_PASSWORD}
+spring.datasource.url=jdbc:postgresql://localhost:5432/green
+spring.datasource.username=postgres
+spring.datasource.password=1111
 spring.datasource.driver-class-name=org.postgresql.Driver
 server.port=0
-
 # Liquibase
-spring.liquibase.url=jdbc:postgresql://localhost:5432/test
-spring.liquibase.user=${DATASOURCE_USER}
-spring.liquibase.password=${DATASOURCE_PASSWORD}
+spring.liquibase.url=jdbc:postgresql://localhost:5432/green
```

Comment:
It's better not to hardcode database configs. Not all of the people in the team might have the same connection parameters